Transbank/Common/RequestService.cs

```diff
@@ -15,13 +15,14 @@ public class RequestService
         private HttpClient _httpClient;
         private static readonly string CONTENT_TYPE = "application/json";
         public RequestService(HttpClient httpClient = null) { _httpClient = httpClient;  }
-        private HttpClient GetHttpClient()
+        private HttpClient GetHttpClient(int requestTimeoutInSeconds)
         {
-            if (_httpClient != null)
-                return _httpClient;
-            if (_staticHttpClient == null)
-                _staticHttpClient = new HttpClient();
-            return _staticHttpClient;
+            _httpClient = new HttpClient
+            {
+                Timeout = TimeSpan.FromSeconds(requestTimeoutInSeconds)
+            };
+
+            return _httpClient;
```

Comment:
Here now ever return a new HttpClient. How does it affect the previous logic that created a new client only if it didn't exist?